React/Views/RCTScrollView.m

```diff
@@ -289,8 +289,9 @@ - (void)dockClosestSectionHeader
   [_stickyHeaderIndices enumerateIndexesWithOptions:0 usingBlock:
    ^(NSUInteger idx, __unused BOOL *stop) {
 
+    // If the subviews are out of sync with the sticky header indices don't
+    // do anything.
     if (idx >= subviewCount) {
-      RCTLogError(@"Sticky header index %zd was outside the range {0, %zd}", idx, subviewCount);
```

Comment:
I'd be fine with that, but still change it to RCTLogWarn, as we're occasionally seeing this error happening with bridgeDidFinishTransaction in the stack trace, so clearly there is at least one scenario where it arises that doesn't relate to scroll event race conditions.